aep/general/0164/aep.md.j2

```diff
@@ -1,5 +1,8 @@
 # Soft delete
 
+**NOTE: this pattern is now deprecated, and is no longer valid. Please use
```

Comment:
We haven't created our first edition, so I don't think we should be "deprecating" AEPs. Let's delete it.